pkg/templates/kubeadm/v1beta4/kubeadm.go

```diff
@@ -0,0 +1,621 @@
+		"DirAvailable--etc-kubernetes-manifests",
+		"ImagePull",
+	}
+
+	clusterConfig := &kubeadmv1beta4.ClusterConfiguration{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "kubeadm.k8s.io/v1beta4",
+			Kind:       "ClusterConfiguration",
+		},
+		ClusterName:          cluster.Name,
+		KubernetesVersion:    cluster.Versions.Kubernetes,
+		ControlPlaneEndpoint: controlPlaneEndpoint,
+		APIServer: kubeadmv1beta4.APIServer{
+			ControlPlaneComponent: kubeadmv1beta4.ControlPlaneComponent{
+				ExtraArgs: []kubeadmv1beta4.Arg{
+					{
+						Name:  "enable-admission-plugins",
+						Value: kubeflags.DefaultAdmissionControllers(),
+					},
+					{
+						Name:  "endpoint-reconciler-type",
+						Value: "lease",
+					},
+					{
+						Name:  "kubelet-certificate-authority",
+						Value: "/etc/kubernetes/pki/ca.crt",
+					},
+					{
+						Name:  "profiling",
+						Value: "false",
+					},
+					{
+						Name:  "request-timeout",
+						Value: "1m",
+					},
+					{
+						Name:  "service-node-port-range",
+						Value: cluster.ClusterNetwork.NodePortRange,
+					},
+					{
+						Name:  "tls-cipher-suites",
+						Value: strings.Join(cluster.TLSCipherSuites.APIServer, ","),
+					},
+				},
+				ExtraVolumes: []kubeadmv1beta4.HostPathMount{},
+			},
+			CertSANs: certSANS,
+		},
+		ControllerManager: kubeadmv1beta4.ControlPlaneComponent{
+			ExtraArgs: []kubeadmv1beta4.Arg{
+				{
+					Name:  "flex-volume-plugin-dir",
+					Value: "/var/lib/kubelet/volumeplugins",
+				},
+				{
+					Name:  "profiling",
+					Value: "false",
+				},
+				{
+					Name:  "terminated-pod-gc-threshold",
+					Value: "1000",
+				},
+			},
+			ExtraVolumes: []kubeadmv1beta4.HostPathMount{},
+		},
+		Scheduler: kubeadmv1beta4.ControlPlaneComponent{
+			ExtraArgs: []kubeadmv1beta4.Arg{
+				{
+					Name:  "profiling",
+					Value: "false",
+				},
+			},
+			ExtraVolumes: []kubeadmv1beta4.HostPathMount{},
+		},
+		Etcd: kubeadmv1beta4.Etcd{
+			Local: &kubeadmv1beta4.LocalEtcd{
+				ImageMeta: kubeadmv1beta4.ImageMeta{
+					ImageRepository: overwriteRegistry,
+				},
+				ExtraArgs: etcdVersionCorruptCheckExtraArgs(cluster.TLSCipherSuites.Etcd),
+			},
+		},
+		DNS: kubeadmv1beta4.DNS{
+			ImageMeta: kubeadmv1beta4.ImageMeta{
+				ImageRepository: defaults(s.Cluster.Features.CoreDNS.ImageRepository, overwriteRegistry),
+			},
+		},
+		ImageRepository: overwriteRegistry,
+		Networking: kubeadmv1beta4.Networking{
+			PodSubnet: join(
+				cluster.ClusterNetwork.IPFamily,
+				cluster.ClusterNetwork.PodSubnet,
+				cluster.ClusterNetwork.PodSubnetIPv6,
+			),
+			ServiceSubnet: join(
+				cluster.ClusterNetwork.IPFamily,
+				cluster.ClusterNetwork.ServiceSubnet,
+				cluster.ClusterNetwork.ServiceSubnetIPv6,
+			),
+			DNSDomain: cluster.ClusterNetwork.ServiceDomainName,
+		},
+	}
+
+	if cluster.ClusterNetwork.KubeProxy != nil && cluster.ClusterNetwork.KubeProxy.SkipInstallation {
+		clusterConfig.DNS.Disabled = cluster.ClusterNetwork.KubeProxy.SkipInstallation
```

Comment:
Can be shortened to `= true`

Reply:
Alternatively, if setting to false would also be fine, remove the second part of the condition instead.